src/gui/sdlmain.cpp

```diff
@@ -2872,6 +2872,10 @@ static void GUI_StartUp(Section *sec)
 	if (!SetDefaultWindowMode())
 		E_Exit("Could not initialize video: %s", SDL_GetError());
 
+	int transparency = section->Get_int("transparency");
```

Comment:
`const auto` (let the `Get_int()` function specify the variable type)

Reply:
suggest clamping `transparency` between 0 and 90: `clamp(section->Get_int("transparency"), 0, 90)`.

Reply:
Thanks for the suggestions. Done both.